src/components/Card.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import PopOverButton from './common/PopOverButton';
+import noneCardImg from '../assets/none-card-img.svg';
+import emptyStar from '../assets/icon-empty-star.svg';
+import fullStar from '../assets/icon-full-star.svg';
+import elapsedTime from '../utils/ElapsedTime';
+import formatDate from '../utils/FormatDate';
+import './Card.css';
+
+const Card = ({ link }) => {
+  const [kebabButtonClick, setKebabButtonClick] = useState(false);
+  const { created_at, createdAt, description, image_source, url } = link;
+  // TODO: 별 체크 기능 추가 예정
+  const fillStar = false;
+
+  return (
+    <div className="card_area">
+      <a
+        href={url}
+        className="card_link"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <div className={`img_area${image_source && ' none_img_area'}`}>
```

Comment:
image_source가 falsy한 경우에는, 예를 들어 undefined인 경우에는 className이 `img_areaundefined`가 되어 스타일이 적용되지 않겠네요!
이런 식으로 삼항연산자로 작성하시면 됩니다.
```suggestion
        <div className={`img_area${image_source ? ' none_img_area' : ''}`}>
```